src/test/java/api-tests/planets_combineconfig_separateFile/combineConfigs.js

```diff
@@ -1,6 +1,7 @@
 function fn() {
     var subConfig = read('../planets-config.js');
-    config = karate.merge(config, subConfig());
-
-    return config;
+    var someOtherConfig = { foo: 'bar' };
+    var mergedConfig = karate.merge(subConfig(), someOtherConfig);
```

Comment:
@Mike-c-Jackson okay, I've added an example of how to use `karate.merge()` here it is

Reply:
Ok, so merging into the original config as noted in the post mentioned at the very beginning, is not valid?

**directed use from first post:**
```
var fun = read('classpath:some.js');
config = karate.merge(config, fun());
```

So no need to merge to the original config as stated, but simply calling the subconfig is sufficient.  If configs can't be merged, I am unclear on the need of the base versus normal config simply making an additional call to a js can add variables at the shared scope level

Reply:
Our big reason to have this second config is because we do a lot of manual API testing in which the environment is not necessarily set through a url call since users are executing steps, based on their account info, during each test run.

Looking to setup this second config to trigger account specific settings, such as username, password, urls, and such based on a default value if noting is passed in a cmd line.  This would allow users to test feature/scenarios manually via the karate:run option rather than having to run a cmd line call for every iteration of a change.

Reply:
> So no need to merge to the original config as stated, but simply calling the subconfig is sufficient. If configs can't be merged, I am unclear on the need of the base versus normal config simply making an additional call to a js can add variables at the shared scope level

The need is explained here: https://stackoverflow.com/a/58339662/143475 - it is for teams that want to do "reuse via a JAR" file, very few teams do this, so you can probably ignore this capability.

Reply:
> Looking to setup this second config to trigger account specific settings, such as username, password, urls, and such based on a default value if noting is passed in a cmd line. This would allow users to test feature/scenarios manually via the karate:run option rather than having to run a cmd line call for every iteration of a change.

This is what environment-specific config is for: https://github.com/karatelabs/karate#environment-specific-config - but you already have the working principle, conditionally read an extra file (or don't)